test/jdk/java/nio/channels/DatagramChannel/Connect.java

```diff
@@ -23,25 +23,36 @@
 
 /* @test
  * @bug 4313882 7183800
+ * @run main/othervm Connect
  * @summary Test DatagramChannel's send and receive methods
  */
 
 import java.io.*;
 import java.net.*;
 import java.nio.*;
 import java.nio.channels.*;
-import java.nio.charset.*;
+import java.time.Instant;
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.CompletionException;
 import java.util.stream.Stream;
 
+import static java.nio.charset.StandardCharsets.US_ASCII;
+
 public class Connect {
 
-    static PrintStream log = System.err;
+    static final PrintStream log = System.err;
+    static final String NOW = Instant.now().toString();
```

Comment:
Would it be possible to rename "log" to "err" so it's a bit clearer at the use-sites where the output is going?

Also I think we should rename "NOW" as it's the time that the class is initialized so very confusing at every usage.

Reply:
done